packages/Utils/use-fork-ref.ts

```diff
@@ -12,8 +9,8 @@ export function setRef(
 }
 
 export function useForkRef(
-  refA: React.MutableRefObject<unknown>,
-  refB: React.MutableRefObject<unknown>
+  refA: ForwardedRef<unknown>,
```

Comment:
changed because it defines an hybrid type for forwarded refs & ref issued by useRef